corehq/apps/api/tests.py

```diff
@@ -1608,6 +1615,36 @@ def test_basic(self):
         self.assert_accessible_via_sessions(self.list_endpoint)
 
 
+class AdminWebResourceTest(APIResourceTest):
+    resource = v0_5.AdminWebUserResource
+    api_name = 'global'
+
+    def tearDown(self):
+        self.community_domain.delete()
+        self.new_user.delete()
```

Comment:
I think if you create these objects in a test it's safer to use `addCleanup` in that same test